src/test/resources/com/washingtonpost/arc/ans/v0_3/model/text-fixture-good-channels.json

```diff
@@ -0,0 +1,6 @@
+{
+    "_id": "0987654321",
```

Comment:
can we add some channels to content_elements in the main story-fixture-good JSON?  that's the JSON we link to from the README.md and that Greg F often asks for as an example of what to use/show?